src/web/mjs/connectors/TsundokuTraducoes.mjs

```diff
@@ -0,0 +1,14 @@
+import WordPressMadaraNovel from './templates/WordPressMadaraNovel.mjs';
+
+export default class TsundokuTraducoes extends WordPressMadaraNovel {
+
+    constructor() {
+        super();
+        super.id = 'tsundokutraducoes';
+        super.label = 'Tsundoku Traduções';
+        this.tags = [ 'webtoon', 'novel', 'portuguese', 'scanlation' ];
+        this.url = 'https://tsundokutraducoes.com.br';
+
+        this.novelContentQuery = 'div.reading-content div[class^="text-"';
+    }
```

Comment:
What about [obstacles removal query](https://github.com/manga-download/hakuneko/blob/c81e82ae03f94e0cd64e760ee3ffcda3bb519cd8/src/web/mjs/connectors/WBNovel.mjs#L12)?
Are there no ads or go-to-top overlays in the novels when reading with HakuNeko that needs to be removed?

Reply:
Updated, see again pls